dashboards-observability/public/components/common/search/autocomplete.tsx

```diff
@@ -20,7 +20,7 @@ import {
   AutocompleteState,
   createAutocomplete
 } from '@algolia/autocomplete-core';
-import { EuiTextArea } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiTextArea } from '@elastic/eui';
```

Comment:
are these used?

Reply:
No they're not, must have snuck in when I was testing them out.